backend/src/util/validate.ts

```diff
@@ -32,7 +34,13 @@ async function calculateBalance(
   const transactionLinkRepository = getCustomRepository(TransactionLinkRepository)
   const { sumHoldAvailableAmount } = await transactionLinkRepository.summary(userId, time)
 
-  if (balance.minus(sumHoldAvailableAmount.toString()).lessThan(0)) {
+  // If we want to redeem a link we need to make sure that the link amount is not considered as blocked
+  // else we cannot redeem links which are more or equal to half of what an account actually owns
+  const sumHoldAvailableAmountMinusTransactionLink = transactionLink
+    ? sumHoldAvailableAmount.minus(transactionLink.amount.toString())
+    : sumHoldAvailableAmount
+
+  if (balance.minus(sumHoldAvailableAmountMinusTransactionLink.toString()).lessThan(0)) {
```

Comment:
```suggestion
  const releasedLinkAmount = transactionLink ? transactionLink.amount : new Decimal(0)

  if (balance.minus(sumHoldAvailableAmount.toString()).plus(releasedLinkAmount.toString()).lessThan(0)) {
```

Reply:
Is it better to release `holdAvailableAmount` or even to calculate the real decay to now?

Reply:
i'd say its better to release the full amount for the link